ci/unit/required_packages.sh

```diff
@@ -24,7 +24,7 @@ pkgs="gotestsum openmpi$OPENMPI_VER                \
       python3$PY_MINOR_VER-junit_xml               \
       python3$PY_MINOR_VER-tabulate numactl        \
       libyaml-devel                                \
-      valgrind-devel patchelf"
+      valgrind-devel patchelf capstone"
```

Comment:
If this is added to the rpm deps now you should not need this here.

Reply:
I wonder if the unit tests run from the docker build rather than rpm?  I thought that had been changed but perhaps not, and this code is also used for NLT which does run from docker.

I know brian has been working on reducing this list but as you say there is still progress to be made.

Reply:
UT is not run from RPMs.